code/modules/mob/living/silicon/ai/freelook/cameranet.dm

```diff
@@ -125,6 +125,8 @@ GLOBAL_DATUM_INIT(cameranet, /datum/cameranet, new)
 /datum/cameranet/proc/majorChunkChange(atom/c, choice)
 	if(!c)
 		return
```

Comment:
I think you should remove these two lines with the advent of your new check. I can't see any reason why calling with a null/already deleted isn't an error

Reply:
fair point
```suggestion
```